app/controllers/coronavirus_form/know_nhs_number_controller.rb

```diff
@@ -21,12 +21,12 @@ def submit
       flash.now[:validation] = invalid_fields
       log_validation_error(invalid_fields)
       render "coronavirus_form/#{PAGE}", status: :unprocessable_entity
+    elsif session[:know_nhs_number] == I18n.t("coronavirus_form.questions.know_nhs_number.options.option_yes.label")
```

Comment:
I wonder if it would be easier to remove the `elsif session["check_answers_seen"]` condition, and just have the else route to `NEXT_PAGE`?

Reply:
There is a scenario where a user edits their answers and selects 'no' (and has seen the 'check your answers' page), so I think we need to keep this condition there.